files/en-us/web/http/headers/want-content-digest/index.html

```diff
@@ -0,0 +1,25 @@
+---
+title: Want-Content-Digest
+slug: Web/HTTP/Headers/Want-Content-Digest
+page-type: http-header
+browser-compat: http.headers.Want-Content-Digest
+spec-urls: https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-digest-headers-12
+status:
+  - experimental
+---
+
+The **`Want-Content-Digest`** request or response header states the wish for a {{HTTPHeader("Content-Digest")}} header.
```

Comment:
Is this really a response header? (asking, not stating). I should check spec, but I have other things on my queue today.

> 
```suggestion
The **`Want-Content-Digest`** request header indicates a desire for the response to include a {{HTTPHeader("Content-Digest")}} header.
```